acceptance/tests/facts/networking_facts.rb

```diff
@@ -11,7 +11,7 @@
 
   agents.each do |agent|
     expected_networking = {
-      %w[networking dhcp] => agent['platform'] =~ /fedora-32|fedora-34|fedora-36|el-8-|el-9-/ ? '' : @ip_regex, # https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/issues/426
+      %w[networking dhcp] => agent['platform'] =~ /fedora-32|fedora-34|fedora-36|fedora-40|el-8-|el-9-/ ? '' : @ip_regex, # https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/issues/426
```

Comment:
I think it would be simpler to replace all of the Fedora versions here with just `fedora-` since this is seemingly an issue with all supported versions of Fedora.

Reply:
Thanks @mhashizume! I have incorporated your suggestion. Please review again.